### PR TITLE
Trying to fix travis

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

**Comment by Ladsgroup** [comment]
The codecov is okay for the first run :D

File: wikiclass/utilities/fetch_labels.py [hunk 1]
```diff
@@ -82,12 +80,8 @@ def aggregate_labels(label_docs):
         return None
     else:
         if sum('automated' not in l['data'] for l in label_docs) > 0:
-            relevant_label_docs = (l for l in label_docs
-                                   if 'automated' not in l['data'])
             auto_labeled = False
         else:
-            relevant_label_docs = (l for l in label_docs
-                                   if 'automated' in l['data'])
```
Comment (halfak) [inline wikiclass/utilities/fetch_labels.py]: Seems like this should not be removed.

Comment (Ladsgroup) [reply wikiclass/utilities/fetch_labels.py]: It's not used at all (unused variable)

Comment (halfak) [reply wikiclass/utilities/fetch_labels.py]: Oh my.  This script is super messed up.  Looks like it was something someone else put together.  I'm merging this but we should fix this script.

**Pull request merged**